### PR TITLE
Added VehLoadout for the British and US

### LINKED ISSUE #35: Add UK vehicle loadouts
Add a vehicle loadout specific to the British.

Use the existing German and Russian vehicle loadouts as an example and use the generic vehicle loadout definitions (found in vehGearGEN.sqf) as much as possible. Furthermore, try to keep the amount of equipment roughly the same as the other vehicle loadouts.

### LINKED ISSUE #37: Add US vehicle loadouts
Add a vehicle loadout specific to the Americans.

Use the existing German and Russian vehicle loadouts as an example and use the generic vehicle loadout definitions (found in vehGearGEN.sqf) as much as possible. Furthermore, try to keep the amount of equipment roughly the same as the other vehicle loadouts.

### PR DESCRIPTION
**Pull request by HeinrichFalk**
Added Vehicle loadouts for UK

This Closes #35 

Added Vehicle loadouts for US

This Closes #37

### CODE REVIEW

File: missions/2PzD_Template.VR/customization/loadouts.sqf [hunk 1]
```diff
@@ -42,8 +42,10 @@
 
 #include "loadoutsVeh\Generic\vehLoadoutGEN.sqf"
```
Comment (HeinrichFalk) [inline missions/2PzD_Template.VR/customization/loadouts.sqf:43]: vehLoadoutGEN needs to be first in this file or the General Geardefs never gets called.

Comment (DarkMessiah27) [reply missions/2PzD_Template.VR/customization/loadouts.sqf:43]: Do you mean first as in, all the way at the top or at the top of the vehicle loadout files?

File: missions/2PzD_Template.VR/customization/loadoutsVeh/British/VehLoadoutUK.sqf [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    [this, UK_Resupply] call Olsen_FW_FNC_VehicleGearScript;         British Resupply Vehicle
+    [this, UK_Squad] call Olsen_FW_FNC_VehicleGearScript;            British Squad Vehicle
+    [this, UK_Section] call Olsen_FW_FNC_VehicleGearScript;          British Section Vehicle
+    [this, UK_Tank] call Olsen_FW_FNC_VehicleGearScript;             British Tank
+*/
+
+//======================== Definitions ========================
+
+#define UK_Infantry_Supplies \
+    [US_Mag_M1T_30, 25] call Olsen_FW_FNC_AddItemVehicle; \
```
Comment (DarkMessiah27) [inline missions/2PzD_Template.VR/customization/loadoutsVeh/British/VehLoadoutUK.sqf:18]: This seems like an awful lot of Thompson ammunition.

File: missions/2PzD_Template.VR/customization/loadoutsVeh/British/VehLoadoutUK.sqf [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+	[UK_Mag_LeeEn, 15] call Olsen_FW_FNC_AddItemVehicle; \
+    [US_Mag_M1919_50_Mixed_Ball, 20] call Olsen_FW_FNC_AddItemVehicle; \
```
Comment (DarkMessiah27) [inline missions/2PzD_Template.VR/customization/loadoutsVeh/British/VehLoadoutUK.sqf:20]: This is twice as much as the Russians and Germans get for their machinegun.

File: missions/2PzD_Template.VR/customization/loadoutsVeh/British/VehLoadoutUK.sqf [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+}];
+
+UK_Squad = ["UK_squad", {
+    params ["_vehicle"];
+
+    GEN_Gren_Smoke;
+    GEN_Flare;
+
+    UK_Infantry_Supplies;
+    UK_Infantry_Supplies;
```
Comment (DarkMessiah27) [inline missions/2PzD_Template.VR/customization/loadoutsVeh/British/VehLoadoutUK.sqf:55]: This only needs one set of Infantry supplies.

File: missions/2PzD_Template.VR/customization/loadoutsVeh/British/VehLoadoutUK.sqf [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+}];
+
+UK_Section = ["UK_Section", {
```
Comment (DarkMessiah27) [inline missions/2PzD_Template.VR/customization/loadoutsVeh/British/VehLoadoutUK.sqf:59]: Section is the British word for Squad. So this should just be Platoon

File: missions/2PzD_Template.VR/customization/loadoutsVeh/British/VehLoadoutUK.sqf [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+    params ["_vehicle"];
+
+    GEN_Gren_Smoke;
+    GEN_Flare;
+    GEN_Medical;
+
+    UK_Infantry_Supplies;
+    UK_Infantry_Supplies;
```
Comment (DarkMessiah27) [inline missions/2PzD_Template.VR/customization/loadoutsVeh/British/VehLoadoutUK.sqf:67]: Again, no need for double infantry supplies

File: missions/2PzD_Template.VR/customization/loadoutsVeh/US/vehLoadoutUS.sqf [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+Tank 1944
+    [this, US_Tank_Late] call Olsen_FW_FNC_VehicleGearScript;        US Tank Late
+*/
+
+//======================== Definitions ========================
+
+#define US_Infantry_Supplies \
+    [US_Mag_M1C, 6] call Olsen_FW_FNC_AddItemVehicle; \
+	[US_Mag_M1T_20, 6] call Olsen_FW_FNC_AddItemVehicle; \
+    [US_Mag_BAR_Mixed_Ball, 20] call Olsen_FW_FNC_AddItemVehicle; \
```
Comment (DarkMessiah27) [inline missions/2PzD_Template.VR/customization/loadoutsVeh/US/vehLoadoutUS.sqf:23]: Might bump this up to 25 even, seeing as the 20rnd magazine is slightly less than half the 50rnd German one.

**Review by DarkMessiah27 — CHANGES_REQUESTED** [review]

**Review by DarkMessiah27 — APPROVED** [review]

**Pull request merged**